libs/external-router/src/transform/resolved-route-to-route.ts

```diff
@@ -23,6 +24,10 @@ export const daffTransformResolvedRouteToRoute = (
     route: {
       path: resolvedRoute.url,
       ...routeType.route,
+      data: {
+        [DAFF_EXTERNAL_ROUTE_DATA_TYPE]: resolvedRoute.type,
+        ...routeType.route.data,
```

Comment:
Do you need to null check the route type route data?

Reply:
`{...null}` is safe and results to `{}` so I don't think so.

Reply:
oh nice, it didn't used to be safe.